pkgs/applications/audio/sublime-music/default.nix

```diff
@@ -19,7 +27,7 @@ python3Packages.buildPythonApplication rec {
 
   nativeBuildInputs = [
     gobject-introspection
-    python3Packages.setuptools
+    python3Packages.poetry
```

Comment:
Can we use poetry-core?

Reply:
I get this error:
```
ModuleNotFoundError: No module named 'poetry.masonry'
```
when I use `poetry-core` instead of `poetry`.

Reply:
You can fetchpatch https://gitlab.com/sublime-music/sublime-music/-/commit/9b0af19dbdfdcc5a0fa23e73bb34c7135a8c2855.patch from https://gitlab.com/sublime-music/sublime-music/-/merge_requests/60